lib/inquire/index.js

```diff
@@ -3,13 +3,24 @@ module.exports = inquire;
 
 /**
  * Requires a module only if available.
+ * All modules requested with inquire must be set as externals in browserify.json, e.g.
+ *   "externals": [
+ *     "long",
+ *     "buffer"
+ *   ]
+ * and added ignored in the package.json browser field, e.g.
+ *     "browser": {
+ *         "long": false,
+ *         "buffer": false
+ *     }
+ *
  * @memberof util
  * @param {string} moduleName Module to require
  * @returns {?Object} Required module if available and not empty, otherwise `null`
  */
 function inquire(moduleName) {
     try {
-        var mod = eval("quire".replace(/^/,"re"))(moduleName); // eslint-disable-line no-eval
+        var mod = require(moduleName);
```

Comment:
The original idea here was to have a way to `require` only if `require` is present (on node), in turn making it necessary that bundlers do not recognize this, yet still having protobuf.js as a dependency somewhere in a graph doesn't blow up. With this change, the common case seems to become that stuff blows up, unless every dependent goes through the hassle of changing their configs.

Reply:
@dcodeIO Yes, as you say, I have kept the original intended behavior by setting the `externals` in the package.json, which creates a trade-off between using eval() and having to set configs.  If that's not acceptable (I understand - it adds maintenance effort), could we create a distribution of ProtobufJS intended for the browsers that does not use eval at all, called `Protobufjs/minimal-browser`?  It seems if you are okay with `inquire` not working in most browsers, then there's no reason to include it.

This should be easy to configure with gulp: we can replace inquire with a dummy function that always returns null.  As I see it, there's no point to calling eval() if it is blocked by the browser, and the message that the browsers print make people think that there is a problem.  What do you think?

Reply:
@dcodeIO Following up on this one more time - I think no config changes are needed by dependent projects. The changes to `package.json` in this repo prevent dependents from blowing up.
I'm sorry I did not clearly address your concern in my last comment. If I can demonstrate that no dependent config changes are needed, would you reconsider this PR?

Reply:
I am mostly out of the loop and it's hard for me to see all the implications. Mostly putting a comment here or there for other reviewers :)